src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -2344,9 +2344,9 @@ LedgerMaster::getFetchPackCacheSize() const
 boost::optional<LedgerIndex>
 LedgerMaster::minSqlSeq()
 {
-#ifdef RIPPLED_REPORTING
     if (app_.config().reporting())
     {
+#ifdef RIPPLED_REPORTING
```

Comment:
Do we report or log an error if reporting is enabled through configuration and the build does not support it?

Reply:
It actually only throws an error in this situation if you are also using Cassandra as a backend. Should probably change that. The best place to detect this and throw would be in either the `ReportingETL` constructor or `ReportingETL::run()`